### PR TITLE
Scrum 510

### PR DESCRIPTION
**Pull request by oblodgett**

### CODE REVIEW

File: src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java [hunk 1]
```diff
@@ -55,7 +55,9 @@ public void createDiseaseAnnotation() {
         diseaseAnnotation.setCurie(DISEASE_ANNOTATION);
         diseaseAnnotation.setNegated(false);
         diseaseAnnotation.setObject(doTerm);
-        diseaseAnnotation.setSubject(biologicalEntity);
+        
+        // TODO fix due to SCRUM-510 refactor
+        //diseaseAnnotation.setSubject(biologicalEntity);
```
Comment (oblodgett) [inline src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java:60]: @abecerra @markquintontulloch  do we need to check on these?

File: src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java [hunk 2]
```diff
@@ -93,7 +95,10 @@ public void editDiseaseAnnotation() {
 
         // change subject
         BiologicalEntity newSubject = createBiologicalEntity("BO:0002", "taxon:0001");
-        editedDiseaseAnnotation.setSubject(newSubject);
+        
+        // TODO fix due to SCRUM-510 refactor
+        //editedDiseaseAnnotation.setSubject(newSubject);
+
```
Comment (oblodgett) [inline src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java:101]: @abecerra @markquintontulloch  do we need to check on these?

**Review by abecerra — APPROVED** [review]

File: src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java (hunk 1 above)
Comment (abecerra) [reply src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java:60]: That Integration test needs a change

File: src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java (hunk 2 above)
Comment (abecerra) [reply src/test/java/org/alliancegenome/curation_api/crud/controllers/DiseaseAnnotationITCase.java:101]: After the conflicts are solved, I will look the edition of negated disease annotations issue in this branch and the edit integration test can be helpful

File: src/main/java/org/alliancegenome/curation_api/base/BaseIdCrudInterface.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+    @POST //@Secured
+    @Path("/")
+    @JsonView(View.FieldsOnly.class)
+    public ObjectResponse<E> create(E entity);
+
+    @GET
+    @Path("/{id}")
+    @JsonView(View.FieldsOnly.class)
+    public ObjectResponse<E> get(@PathParam("curie") Long id);
```
Comment (oblodgett) [inline src/main/java/org/alliancegenome/curation_api/base/BaseIdCrudInterface.java:27]: This should be "id"

**Review by abecerra — APPROVED** [review]

**Pull request merged**